R/data_kara.R

```diff
@@ -0,0 +1,49 @@
+#' NBA Elo Ratings Through History
+#' 
+#' The data behind the Elo and RAPTOR ratings for NBA teams used in various stories like this one: 
```

Comment:
We have been using a consistent standard for the description of all datasets, could you please make this match too? Here is an example: https://github.com/rudeboybert/fivethirtyeight/blob/master/R/data_anna.R#L3

Reply:
Could you make all the fixes needed in this help file?